app/assets/stylesheets/media.scss

```diff
@@ -331,6 +330,14 @@
       padding: 0.2rem 0.4rem;
       text-transform: uppercase;
     }
+
+    button.dismissButton {
+      padding: 0rem;
+
+      svg {
+        font-size: 0.85rem;
+      }
+    }
```

Comment:
this styling was just the result of a bit of tweaking till i got something that i thought looked slightly better than the default.  very open to suggestions here.